src/core/Akka.NodeTestRunner/Program.cs

```diff
@@ -36,7 +51,7 @@ static void Main(string[] args)
                 Console.WriteLine("Running specs for {0} [{1}]", assemblyName, assemblyFileName);
                 using (var discovery = new Discovery(assemblyName, typeName))
                 {
-                    using (var sink = new Sink(nodeIndex))
+                    using (var sink = new Sink(nodeIndex, tcpClient))
```

Comment:
the `Sink` now takes a directly reference to the `RunnerTcpClient` actor reference and writes to that.